gallery_dl/extractor/bilibili.py

```diff
@@ -81,6 +81,31 @@ def items(self):
             yield Message.Url, url, text.nameext_from_url(url, article)
 
 
+class BilibiliUserArticleFavlistExtractor(BilibiliExtractor):
+    subcategory = "user-article-favlist"
+    pattern = (r"(?:https?://)?(?:space.bilibili.com)"
+               r"/(\d+)/(?:favlist\?fid=opus)"
+               r"(?:&ftype=opus)?")
+    example = "https://space.bilibili.com/123456/favlist?fid=opus"
```

Comment:
```suggestion
    example = "https://space.bilibili.com/12345/favlist?fid=opus"
```

Example URL IDs are `12345` everywhere else.